src/modules/reactions.js

```diff
@@ -0,0 +1,23 @@
+const sample = require('lodash/sample');
+const { gifs } = require('./reaction-list.json');
+const { SOCIAL_PREFIX } = require('../config');
+
+const reactionSocial = async function (response, { /* contex, */ input, i18n }) {
+    if (input[0] !== SOCIAL_PREFIX) {
+        return response;
+    }
+    // const { id } = contex;
+
```

Comment:
- [ ] u need keep "allowed" reactions, and if input start from prefix BUT dont allowed throw error